adversarial-nn/adversarial-nn.json

```
@@ -0,0 +1,36 @@
+{
+    "key": "adversarial-nn"
```

```suggestion
    "key": "adversarialnn",
```

Missing trailing comma, `key` cannot contain a hyphen